cli/cli.ts

```diff
@@ -993,12 +994,15 @@ function uploadCoreAsync(opts: UploadOptions) {
                 }
 
                 if (minified) {
-                    const res = uglify.minify(content);
-                    if (!res.error) {
-                        content = res.code;
-                    }
-                    else {
-                        pxt.log(`        Could not minify ${fileName} ${res.error}`)
+                    if (mime == "application/javascript") {
+                        const res = uglify.minify(content);
```

Comment:
Are there options we can pass here. @dragonx Do you know if uglifycss removes comments as well?